hay-espacio-en-el-taco/dbs-scraper
# First working version with react-window

## What's the purpose of this PR?
Instead of an infinite scroll it uses the [react-window](https://github.com/bvaughn/react-window) module.

This should improve the performance and also allows to "show" the actual amount of cards.

## How can this be tested?
* Run the app.
* Confirm that all the cards render correctly.
* Confirm that scrolling is a smooth experience and consistent no matter how many cards are shown.
